jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/composite/MaterialSelectorContainer.java

```diff
@@ -1,216 +1,216 @@
+        for (int i = 0; i < groupSize; i++) {
+            String stringGroupsAndOptions = webList.values().get(i);
+            List<String> listGroupsAndOptions = Stream.of(stringGroupsAndOptions.split("\n"))
+                    .collect(Collectors.toCollection(ArrayList::new));
+            groups.add(listGroupsAndOptions.get(0));
+        }
+        collapsePanel();
+        return groups;
+    }
+
+    /**
+     * Get the available selector groups and options.
+     *
+     * @return Map<String, List < String>> map of available groups and options
+     */
+    @JDIAction("Get '{name}' groups and options")
+    public Map<String, List<String>> getMapGroupsAndOptions() {
+        Map<String, List<String>> map = new LinkedHashMap<>();
+        WebList webList = new WebList(By.cssSelector(groupsAndOptionsList));
+        int groupSize = webList.values().size();
+        for (int i = 0; i < groupSize; i++) {
+            String stringGroupsAndOptions = webList.values().get(i);
+            List<String> listGroupsAndOptions = Stream.of(stringGroupsAndOptions.split("\n"))
+                    .collect(Collectors.toCollection(ArrayList::new));
+            String key = listGroupsAndOptions.remove(0);
+            map.put(key, listGroupsAndOptions);
+        }
+        collapsePanel();
+        return map;
+    }
+
+    /**
+     * Get the list of enabled elements.
+     *
+     * @return List&lt;String&gt; enabled values
+     */
+    @JDIAction("Get '{name}' enabled values")
+    public List<String> listEnabled() {
+        List<String> list = getOptions(matOptions).stream()
+                .filter(e -> !e.hasAttribute("disabled") && e.attr("aria-disabled").contains("false"))
+                .map(e -> e.text().trim()).collect(Collectors.toList());
+        collapsePanel();
+        return list;
+    }
+
+    /**
+     * Get the list of disabled elements.
+     *
+     * @return List&lt;String&gt; disabled values
+     */
+    @JDIAction("Get '{name}' disabled values")
+    public List<String> listDisabled() {
+        List<String> list = getOptions(matOptions).stream()
+                .filter(e -> e.hasAttribute("disabled") || e.attr("aria-disabled").contains("true"))
+                .map(e -> e.text().trim()).collect(Collectors.toList());
+        collapsePanel();
+        return list;
+    }
+
+    /**
+     * Check the element background has a color with the specified parameters.
+     *
+     * @param red   decimal representation of color Red of the form rgba(R,G,B,a)
+     * @param green decimal representation of color Green of the form rgba(R,G,B,a)
+     * @param blue  decimal representation of color Blue of the form rgba(R,G,B,a)
+     * @param a     opacity of the color of the form rgba(R,G,B,a)
+     * @return boolean is background has a specified color
+     */
+    @JDIAction("Check that rgba({0}, {1}, {2}, {3}) is the specified color")
+    public boolean color(final int red, final int green, final int blue, final double a) {
```

Comment:
Color validation is not a good function for class, it is better to have it in assertion